devel/irrlichtmt/Portfile

```diff
@@ -19,3 +20,29 @@ checksums           rmd160  4da410c24f67e8c0ebd9880308b8d796d2f86676 \
 depends_lib         path:include/turbojpeg.h:libjpeg-turbo \
                     port:libpng \
                     port:zlib
+
+pre-build {
+
+# Fix01: irrlichtmt fails to install on Mac OS 10.6 <https://trac.macports.org/ticket/66439>
+# Add <GL/glext.h> - OpenGL 1.2 and above compatibility profile and extension interfaces.
+#
+# Copied from: <https://registry.khronos.org/OpenGL/index_gl.php#headers>
+# from section 'API and Extension Header Files' as of 2023-01-07
+
+    if {${os.major} <= 10} {
+        xinstall ${filesdir}/fix01-glext.h ${worksrcpath}/include/glext.h
```

Comment:
Did you consider adding a dependency on the `mesa` port? It seems to have the `GL/glext.h` header.

Reply:
Aha? IMO better late than never.

I will consider this in due time.